tests/verify-pool-member-vdo.yml

```diff
@@ -0,0 +1,38 @@
+- name: check VDO
+  block:
+  - name: get information about VDO deduplication
```

Comment:
The tasks inside the block need to be indented by an additional two spaces - so
```yaml
  block:
    - name: get information about VDO deduplication
      command: "vdostats --verbose {{ storage_test_pool.name }}-vdopool-vpool"
...
```

Reply:
Fixed.